packaging/centos-7/build-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-7:latest
+MAINTAINER support@osuosl.org
+RUN yum -y install \
+    gcc \
+    gcc-c++ \
+    glibc \
+    glibc-devel \
+    glibc-static \
+    libstdc++-static \
+    make \
+    tar \
+    wget \
+    zlib \
+    zlib-devel \
+    zlib-static \
+    && yum clean all
+RUN mkdir /dssp
+WORKDIR /dssp
+RUN wget http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz && \
+    tar zxf bzip2-1.0.6.tar.gz
+WORKDIR /dssp/bzip2-1.0.6
+RUN mv Makefile Makefile.old && sed -e "/^CFLAGS/s/$/ -fPIC/" < Makefile.old > Makefile && make
+RUN make install
+WORKDIR /dssp
+RUN rm -rf bzip2-1.0.6 bzip2-1.0.6.tar.gz
+RUN wget http://downloads.sourceforge.net/project/boost/boost/1.57.0/boost_1_57_0.tar.gz && \
```

Comment:
https://github.com/osuosl/dssp/pull/3#discussion_r69950956

Because I need the static libraries.